### PR TITLE
Fixing endpoint displays, ref. DCOS-8975

### PR DESCRIPTION
**Pull request by mlunoe**
BRIDGE network:

HOST network:

Virtual Networks is still missing

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @jfurrow

**Comment by mlunoe** [comment]
@Poltergeist Would be nice if I could get your eyes on this before tomorrow morning

**Comment by mlunoe** [comment]
Tests for the `TaskEndpointsList` are failing, will fix tomorrow.

**Comment by Poltergeist** [comment]
**mesosphere-ci** retest this please

**Comment by Poltergeist** [comment]
It looks good to me and works very fine! Sorry for the delay.

File: src/js/utils/TaskUtil.js [hunk 1]
```diff
@@ -74,6 +85,29 @@ const TaskUtil = {
     }
 
     return portMappings;
+  },
+
+  getNetwork(task) {
```
Comment (jfurrow) [inline src/js/utils/TaskUtil.js]: What exactly does this return? Perhaps it could be called `#getNetworkType`?

File: src/js/utils/TaskUtil.js [hunk 2]
```diff
@@ -2,42 +2,53 @@
...
+    let ports = this.getPorts(task);
+    let hostName = node.getHostName();
+
+    // networkType is 'HOST', but no host name
+    if (networkType !== 'BRIDGE' && networkType !== 'USER' && !hostName) {
+      return {ports: [], hosts: []};
+    }
+
+    // networkType is 'HOST'
+    if (networkType !== 'BRIDGE' && networkType !== 'USER') {
```
Comment (jfurrow) [inline src/js/utils/TaskUtil.js]: Do we need these negative checks? Can we not just use `networkType === 'HOST'`? Just curious, I don't know the details here...

Comment (mlunoe) [inline src/js/utils/TaskUtil.js]: This is future proofing, and defaulting it to handle it as 'HOST' network, so if they ever add another network type, we will handle it as 'HOST' by default

**Pull request merged**